sections/shared/Layout/AppLayout/Footer/Footer.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+
+const Footer: FC = () => {
```

Comment:
I think it might be better to call this `GitID` and respectively `GitID.tsx`. It's a bit confusing since there is already another Footer component.